dashboard/test/ui/features/star_labs/gamelab/export_animations.feature

```diff
@@ -1,7 +1,7 @@
 @as_student
 Feature: Game Lab Export
 
-@no_mobile
+@no_mobile @no_safari
```

Comment:
Might be worth adding a comment about why Safari is skipped here.

Reply:
Agreed. Do you know how I add a comment to a .feature file? I'm not seeing any examples in this or nearby files.

Reply:
I think it's `#`